pluginDefinition.json

```diff
@@ -25,7 +25,8 @@
       "version": "1.0.0",
       "initializerLookupMethod": "external",
       "initializerName": "ipExplorerDataServiceInstaller",
-      "libraryName": "ipExplorer.so",
+      "libraryName31": "ipExplorer31.so",
```

Comment:
Please also keep `"libraryName": "ipExplorer31.so",` because it seems this plugin works in both v1 and v2, so it could be possible to keep it that way.

Reply:
Resolved